IP Table resource: Added option to ignore rule comments (#5777)
chef/inspec: docs-chef-io/content/inspec/resources/iptables.md

- `rule:'name'` is the name of a rule that matches a set of packets
- `table:'name'` is the packet matching table against which the test is run
- `chain: 'name'` is the name of a user-defined chain or one of `ACCEPT`, `DROP`, `QUEUE`, or `RETURN`

> - `ignore_comments: true` is the boolean flag used when comment in the rule needs to be ignored.

Reviewer: ```suggestion
- `ignore_comments: true` is a boolean flag that ignores comments in a rule.
```
